src/Angular.js

```diff
@@ -146,13 +146,13 @@ var uppercase = function(string) {return isString(string) ? string.toUpperCase()
 
 
 var manualLowercase = function(s) {
-  /* jshint bitwise: false */
+  /* JSHint bitwise: false */
```

Comment:
This capitalization of `JSHint` is not recognized and the code below generates a lint error. Same below